From a1ba968f8f93adf2922f7e84af1213e6fde5363c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Diego=20Elio=20'Flameeyes'=20Petten=C3=B2?= <flameeyes@gmail.com>
Date: Thu, 29 Jul 2010 01:48:24 +0200
Subject: [PATCH] If possible, try to load simply spec_helper, the Rakefile will take care.

If the file is loaded with relative paths it won't be able to load it
properly.
---
 spec/hashie/clash_spec.rb |    2 +-
 spec/hashie/dash_spec.rb  |    2 +-
 spec/hashie/hash_spec.rb  |    2 +-
 spec/hashie/mash_spec.rb  |    2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/spec/hashie/clash_spec.rb b/spec/hashie/clash_spec.rb
index 6363d5d..8a8a909 100644
--- a/spec/hashie/clash_spec.rb
+++ b/spec/hashie/clash_spec.rb
@@ -1,4 +1,4 @@
-require File.dirname(__FILE__) + '/../spec_helper'
+require 'spec_helper' rescue require File.dirname(__FILE__) + '/../spec_helper'
 
 describe Hashie::Clash do
   before do
diff --git a/spec/hashie/dash_spec.rb b/spec/hashie/dash_spec.rb
index b7971f9..8a6d466 100644
--- a/spec/hashie/dash_spec.rb
+++ b/spec/hashie/dash_spec.rb
@@ -1,4 +1,4 @@
-require File.dirname(__FILE__) + '/../spec_helper'
+require 'spec_helper' rescue require File.dirname(__FILE__) + '/../spec_helper'
 
 class DashTest < Hashie::Dash
   property :first_name
diff --git a/spec/hashie/hash_spec.rb b/spec/hashie/hash_spec.rb
index 78d44be..69e43b5 100644
--- a/spec/hashie/hash_spec.rb
+++ b/spec/hashie/hash_spec.rb
@@ -1,4 +1,4 @@
-require File.dirname(__FILE__) + '/../spec_helper'
+require 'spec_helper' rescue require File.dirname(__FILE__) + '/../spec_helper'
 
 describe Hash do
   it "should be convertible to a Hashie::Mash" do
diff --git a/spec/hashie/mash_spec.rb b/spec/hashie/mash_spec.rb
index 70d1f6f..080ade6 100644
--- a/spec/hashie/mash_spec.rb
+++ b/spec/hashie/mash_spec.rb
@@ -1,4 +1,4 @@
-require File.dirname(__FILE__) + '/../spec_helper'
+require 'spec_helper' rescue require File.dirname(__FILE__) + '/../spec_helper'
 
 describe Hashie::Mash do
   before(:each) do
-- 
1.7.2

